src/tools/interop/idt/scripts/push.sh

```diff
@@ -0,0 +1,21 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+source idt/scripts/vars.sh
+mv idt/venv TEMPvenv
```

Comment:
Try using `mktemp -d` for temporary file in the future. After creating a temporary file you can do something like this to make sure the file is cleaned up, even on error.

```bash
tmpfile=$(mktemp -d /tmp/idt.XXX) && trap "rm -rf $TMPFILE" EXIT ERR
```

Reply:
Thanks!